LandWeb_preamble.R

```diff
@@ -176,6 +182,12 @@ Init <- function(sim) {
     ml <- provNWT(ml, P(sim)$runName, dataDir, sim$canProvs, P(sim)$bufferDist, asStudyArea = TRUE)
   } else if (grepl("provSK", P(sim)$runName)) {
     ml <- provSK(ml, P(sim)$runName, dataDir, sim$canProvs, P(sim)$bufferDist, asStudyArea = TRUE)
+  } else if (grepl("Smoky", P(sim)$runName)) {
+    ml <- processPSR(ml, config::get('studyarea'), dataDir, sim$canProvs, P(sim)$bufferDist, asStudyArea = TRUE)
+  } else if (grepl("Bistcho", P(sim)$runName)) {
+    ml <- processPSR(ml, config::get('studyarea'), dataDir, sim$canProvs, P(sim)$bufferDist, asStudyArea = TRUE)
+  } else if (grepl("Cold", P(sim)$runName)) {
+    ml <- processPSR(ml, config::get('studyarea'), dataDir, sim$canProvs, P(sim)$bufferDist, asStudyArea = TRUE)
```

Comment:
these can be handled with single `grepl` call since they are the same lines of code.

NOTE: do NOT use `config::get()` anywhere except the `01-init.R` script because you will have to deal with race conditions.

Is there a reason you aren't using `runName` mechanism to pass the study area name?

Reply:
i suggest using PSR<blah> in the runName, so we grep `"PSR"` at tho top level here, and deal with the specific PSR (i.e., `<blah>`) in `processPSR()` -- see FMU code